doc/reference.md

```diff
@@ -272,10 +326,14 @@ and serializable.
 
 Represents the non-confidential details of a payment card.
 
-Field          | Type   | Description
--------------- | ------ | -----------------
-brand          | String | [CardBrand](#cardbrand) constant that indicates the entity responsible for issuing the card.
-lastFourDigits | String | Indicates how the card information was captured.
+Field             | Type            | Description
+----------------- | --------------- | -----------------
+brand             | String          | [CardBrand](#cardbrand) constant that indicates the entity responsible for issuing the card.
+lastFourDigits    | String          | Indicates how the card information was captured.
+expirationMonth   | integer         | The month of the card’s expiration date, if available. This value is always between 1 and 12, inclusive.
+expirationYear    | integer         | The four-digit year of the card’s expiration date, if available.
```

Comment:
Style issue: avoid contractions. Suggest: 

The 4-digit year  when the associated card expires, if available.